docs/resources/aviatrix_transit_gateway.md

```diff
@@ -303,33 +303,33 @@ resource "aviatrix_transit_gateway" "test-edge-transit-1" {
     site_id = "site-3"
     interfaces {
         gateway_ip    = "192.168.20.1"
-        name          = "eth0"
         ip_address    = "192.168.20.11/24"
         type          = "WAN"
+        index         = 0
```

Comment:
Should there also be aep documentation changes?

Reply:
AEP documentation change wont be required here.